.github/workflows/wheels.yaml

```
@@ -60,6 +62,13 @@ jobs:
       - name: Build sdist
         run: pipx run build --sdist
 
+      - name: Test sdist
+        run: |
+          python -m pip install 'dist/*.tar.gz[test]'
```

Is this what you're going for here? The `[test]` suffix breaks the shell's glob expansion, so we need to wrap it in a command substitution.
```suggestion
          python -m pip install "$(echo dist/*.tar.gz)[test]"
```

---

thanks a lot !